### PR TITLE
Update asset client to incorporate Pike Agents

### PR DESCRIPTION
**Pull request by dnewh**
Updates the Track and Trace Asset Client to use Pike Agents rather than the Track and Trace agents.

### CODE REVIEW

File: track_and_trace/asset_client/src/views/signup_form.js [hunk 1]
```diff
@@ -23,84 +23,197 @@ const forms = require('../components/forms')
...
+                      onclick: AgentSignUp.submit,
+                      disabled: AgentSignUp.submitting || AgentSignUp.invalid(),
+                    }, 'Create Agent'))))
+        ])
+    ]
   }
 }
 
-module.exports = SignupForm
+// const passwordCard = state => {
```
Comment (agunde406) [inline track_and_trace/asset_client/src/views/signup_form.js]: Please do not include commented out code in your commits.

Comment (dnewh) [reply track_and_trace/asset_client/src/views/signup_form.js]: Commented block removed

File: track_and_trace/asset_client/src/views/list_organizations.js [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+const m = require('mithril')
+const sortBy = require('lodash/sortBy')
+const truncate = require('lodash/truncate')
+const {Table, FilterGroup, PagingButtons} = require('../components/tables.js')
+const api = require('../services/api')
+const organizations = require('../services/organizations')
+
+const PAGE_SIZE = 50
+
+const OrganizationList = {
```
Comment (peterschwarz) [inline track_and_trace/asset_client/src/views/list_organizations.js]: There's a lot of new things in the PR, that perhaps could be broken into more commits.

Comment (peterschwarz) [reply track_and_trace/asset_client/src/views/list_organizations.js]: Also, more detail in the commit messages would be a good change.

**Pull request merged**